features/adresregels.feature

```diff
@@ -0,0 +1,133 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik samengestelde adresregels
+    zodat ik het samengestelde adres direct en correct kan gebruiken.
+
+    Het adres wordt samengesteld conform NEN 5825:2002. Dit betreft alleen het adresdeel van de adressering, het naamdeel van de adressering komt uit een andere bron (zoals BRP, HR of BRK).
+
+    De adresregels worden samengesteld op basis van de volgende regels
+    - Property adresregel1 wordt samengesteld uit korteNaam + huisnummer + huisletter + huisnummertoevoeging
+    - Property adresregel2 wordt samengesteld uit postcode + woonplaats
+    - Tussen kortenaam en huisnummer wordt een spatie opgenomen
+    - Tussen het huisnummer en de huisnummertoevoeging (wat in de NEN een samenstelling is van huisletter en huisnummertoevoeging, niet gecheiden van elkaar) moet een koppelteken ("-") worden geplaatst indien de huisnummertoevoeging met een cijfer begint, een spatie in alle andere gevallen
+    - Tussen postcode numeriek deel en postcode alfabetisch deel moet een spatie worden geplaatst
+    - Tussen postcode alfabetisch deel en woonplaatsnaam moeten twee spaties worden geplaatst
+
+    Scenario: er is geen huisnummer en geen huisletter
```

Comment:
Er staat geen huisnummer, maar in het gegeven is er wel een huisnummer

Reply:
moet zijn "er is geen huisnummertoevoeging"